sktime/forecasting/tests/test_naive.py

```diff
@@ -172,3 +172,14 @@ def test_strategy_drift_window_length(fh, window_length):
         expected = values[-1] + slope * fh
 
         np.testing.assert_array_equal(y_pred, expected)
+
+
+@pytest.mark.parametrize("fh", TEST_OOS_FHS)
+def test_return_pred_int(fh):
```

Comment:
can we add a test to compare against results from R?

Reply:
Yes I think, for now, I'll skip this I'll make this change in a future Commit